todocore/src/main/java/com/knighttodo/todocore/rest/DayTodoResource.java

```diff
@@ -175,4 +178,26 @@ public DayTodoReadyResponseDto updateIsReady(@PathVariable UUID dayId, @PathVari
             throw new DayTodoReadyCanNotBeUpdatedException("Day todo ready can't be updated.", ex);
         }
     }
+
+    @PatchMapping("")
+    @ResponseStatus(HttpStatus.OK)
+    @ApiOperation(value = "Change orderNumber id existed Todo", response = DayTodoResponseDto.class, responseContainer = "List")
```

Comment:
Change orderNumber id in existing Todo looks better, use Google translate or MS Word to check spelling, my browser checks it.

Reply:
Use Grammarly plugin for the browser and you can use it online.

Reply:
Done.